source/blog/2018-04-13-ember-3-1-released.md

```diff
@@ -200,7 +210,7 @@ In Ember 3.1 we've corrected the implementation to act like a proper closure. In
 For more information about this change see
 [emberjs/ember.js#15287](https://github.com/emberjs/ember.js/pull/15287).
 
-### Deprecations in Ember 3.1
+### Deprecations in Ember 3.1 (2)
```

Comment:
this is confusing for me :/

Reply:
ok i'll just remove it for now. I have a template worked out for future posts so we get the consistency we're after.